scripts/token-hud-extension.js

```diff
@@ -56,7 +60,7 @@ class TokenHudExtension {
             }
             if (ev.altKey && ev.shiftKey) {
                 // TODO: Interrogate actor Ride specializations and offer selection if more than one is available
-                if (hasSkill(actor, "Ride") !== null) {
+                if (hasSkill(actor, "Ride (Horse)") !== null) {
```

Comment:
Ride (Horse) is most common Ride specialisation and a good default, but why not check for Grouped now that functionality is provided?   I may have misunderstood how you intended that functionality to behave.